source/Nevermore/Advanced/ReadTransaction.cs

```diff
@@ -665,6 +666,38 @@ protected async Task<TResult[]> ReadResultsAsync<TResult>(PreparedCommand prepar
             using var command = CreateCommand(preparedCommand);
             return await command.ReadResultsAsync(mapper, cancellationToken).ConfigureAwait(false);
         }
+        
+        async Task<DbTransaction> BeginTransactionWithRetryAsync(IsolationLevel isolationLevel, string sqlServerTransactionName, RetryPolicy retryPolicy)
```

Comment:
These moved out of the now-deleted `TransactionExtensions` in order to allow reassignment of `connection`.
The extensions weren't used anywhere else and arguably it's better to have them as private helpers here anyway